processes/ingest/chicago_isac.py

```diff
@@ -72,18 +79,22 @@ def storePDFManifest(self, record):
                     'application/webpub+json',
                     flags
                 ])
+
                 record.has_part.insert(0, linkString)
-                #The second element of the has_part array will always be an array of pdfURLS from the json file
-                #Those elements of the array are popped out and appended back into the has_part array as strings instead of an array
-                for i in range(1, len(record.has_part)):
-                    if len(record.has_part[i]) > 1:
-                        urlArray = record.has_part[i]
-                        record.has_part.pop(i)
-                        for elem in urlArray:
-                            record.has_part.append(elem)
-                    else:
-                        record.has_part[i] = ''.join(record.has_part[i])
+                self.hasPartArrayElemToStringElem(record)
+
                 break
+    
+    @staticmethod
+    def hasPartArrayElemToStringElem(record):
```

Comment:
this method name is a bit convoluted. could we change this name to be clearer?

Reply:
I can change the name to `change_has_part_url_array_to_string`